app/javascript/react/components/GiraffesContainer.js

```diff
@@ -28,6 +29,7 @@ const GiraffesContainer = (props) => {
 
   return (
     <div className="grid-container">
+      <Link to="/giraffes/new">Add a Giraffe</Link>
```

Comment:
Maybe don't need this since we have the nav bar in our branch?

Reply:
true, removed link